module.txt

```diff
@@ -5,7 +5,8 @@
     "displayName" : "Climate Conditions",
     "description" : "Module that introduces humidity and temperature fields and their modifiers",
     "dependencies" : [
-        {"id" : "Core", "minVersion" : "3.0.0"}
+        {"id" : "Core", "minVersion" : "3.0.0"},
+        {"id" : "Thirst", "minVersion" : "1.0.0"}
```

Comment:
I'd propose extracting the desert effect into a separate PR. Thus you don't require the Thirst dependency in this PR.